pyproject.toml

```diff
@@ -0,0 +1,63 @@
+[project]
+name = "rocketpy"
+version = "1.2.2"
+description="Advanced 6-DOF trajectory simulation for High-Power Rocketry."
+dynamic = ["dependencies", "optional-dependencies"]
+readme = "README.md"
+requires-python = ">=3.8"
+authors = [
+  {name = "Giovani Hidalgo Ceotto", email = "ghceotto@gmail.com"}
+]
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "License :: OSI Approved :: MIT License",
+    "Operating System :: OS Independent",
+    "Topic :: Rocket Flight Simulation :: Libraries :: Python Modules",
+]
+
+[project.urls]
+homepage = "https://rocketpy.org/"
+documentation = "https://docs.rocketpy.org/"
+repository = "https://github.com/RocketPy-Team/RocketPy"
+
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[tool.setuptools]
+py-modules = []
+
+[tool.setuptools.package-data]
+drag_curves = ["**/*.csv"]
+engine_files = ["**/*.eng"]
+environment_files = ["**/*.nc"]
+jupyter_notebooks = ["**/*.ipynb"]
+documentation = ["**/*.rst"]
+
+[tool.setuptools.dynamic]
+dependencies = {file = ["requirements.txt"]}
+optional-dependencies.env_analysis = {file = ["requirements-optional.txt"]}
+optional-dependencies.test = {file = ["requirements-tests.txt"]}
+
+[tool.black]
+line-length = 88 
+include = '\.py$'
+skip-string-normalization = true
+
+[tool.pylint]
+max-line-length = 88 
+disable = """
+    line-too-long,
+    duplicate-code,
+    use-dict-literal,
+    missing-module-docstring,
+    missing-function-docstring,
+    missing-class-docstring,
+    too-few-public-methods,
+    too-many-public-methods,
+    too-many-instance-attributes,
+    logging-fstring-interpolation,
+    broad-exception-raised,
+    import-error,
+    protected-access
+"""
```

Comment:
Where did you get these rules from? The .pylintrc file already has different rules to customize pylint to our project. I know that most of that file is composed by boilerplates and are not actually being used, so we get two different options then: (1) not defining pylint rules in the toml project and keep using the .pylintrc file or (2) select only the rules that are actually being used in the .pylintrc file.

```
[tool.pylint]
max-line-length = 88 
disable = """
    line-too-long,
    duplicate-code,
    use-dict-literal,
    missing-module-docstring,
    missing-function-docstring,
    missing-class-docstring,
    too-few-public-methods,
    too-many-public-methods,
    too-many-instance-attributes,
    logging-fstring-interpolation,
    broad-exception-raised,
    import-error,
    protected-access
"""
```

Reply:
Sorry! I didn't notice that file. 

From the two I prefer the (2), but instead of listing all enabled I'll just list the disabled rules here and everything else is understood as enabled.